components/net/mime_classifier.rs

```diff
@@ -196,7 +196,8 @@ impl MimeClassifier {
 
     fn is_xml(mt: &Mime) -> bool {
         mt.suffix() == Some(mime::XML) ||
-        mt.type_() == mime::APPLICATION && mt.subtype() == mime::XML
+        mt.type_() == mime::APPLICATION && mt.subtype() == mime::XML ||
+        mt.type_() == mime::TEXT && mt.subtype() == mime::XML
```

Comment:
Could we add parentheses to this? I can never remember the precedence of || vs &&.